felix/bpf/ut/pol_prog_test.go

```diff
@@ -300,6 +338,37 @@ var polProgramTests = []polProgramTest{
 			tcpPkt("10.0.0.2:80", "10.0.0.1:31245"),
 			icmpPkt("10.0.0.1", "10.0.0.2")},
 	},
+	{
+		PolicyName: "unreachable tier-v6",
```

Comment:
I was wondering if it was just possible to remove the hardcoded strings, replace them with variables and parametrize the tests with two sets of IPs so that we would have the same set for both 4/6 and when we add a test, we would automatically add that test for both :thinking:

Reply:
It's possible, but I think in the end we would need to specify both IPv4/6 rules and packets separately, which does not shorten the test cases. Also for IPv6 CIDR tests I had to add more tests to cover situation about mask length greater and shorter than 64 which is not valid for IPv4. Do you think I should try it?